### ISSUE TITLE
Get Current User Password

### ISSUE THREAD

**Issue by stixoffire** [volkanceylan/Serenity#4326]
I am using Active Directory for User Sign on ; and I am wanting to send an email out when the current user commits a New xyz row.  I can get username and there is also password hash , anyway to return the hash back into a PW for an email smtp server.

**Comment by ga5tan** [comment]
There should be no way to decode password, cause otherwise it would be a security breach. I hope we are using proper encryption and it should not be possible to decode it.

**Comment by stixoffire** [comment]
@ga5tan I understand the security issue here .. however I would like the system to send out a notification

**Comment by stixoffire** [comment]
For anyone else looking for a simple answer - I created a separate special email account.
My MailKit authenticated with my new user name / password (which unfortunately is hard-coded ).
However when sending I tried to Message.From.Add(myCurrentUser)  this refused to send the message .. So I had to add my special account instead of the user that created the New Record. I then added that user in message Subject / Body .. not ideal but it works. Also since my email to comes from db I had to ensure that I had a real address to add to from / cc: ...
